test/helpers.tcl

```diff
@@ -203,18 +203,16 @@ suppress_message ODB 127
 # Finished DEF file
 suppress_message ODB 134
 
-# suppress ppl info messages. The ones defined in tcl can never
-# match between tcl and Python
+# suppress ppl info messages which are only present in tcl
 suppress_message PPL 41
-suppress_message PPL 48
-suppress_message PPL 67
 suppress_message PPL 60
+suppress_message PPL 67
```

Comment:
There appears to be no message 67.

Reply:
Seems there are when removing the suppression, here an example:
```
** /home/titan/OpenROAD/src/ppl/test/add_constraint1.ok	2024-10-12 13:35:34.649208359 +0200
--- /home/titan/OpenROAD/src/ppl/test/results/add_constraint1-tcl.log	2024-10-26 17:10:29.250298762 +0200
***************
*** 3,8 ****
--- 3,9 ----
  [INFO ODB-0130]     Created 54 pins.
  [INFO ODB-0131]     Created 88 components and 422 component-terminals.
  [INFO ODB-0133]     Created 54 nets and 88 connections.
+ [INFO PPL-0067] Restrict INPUT pins to region 0.0u-100.13u, in the top edge.
  Found 0 macro blocks.
  [INFO PPL-0001] Number of slots           2494
  [INFO PPL-0002] Number of I/O             54
```